lib/pages/new_group/new_group.dart

```diff
@@ -168,6 +168,21 @@ class NewGroupController extends State<NewGroup>
     );
   }
 
+  void createNewGroup({ String? urlAvatar }) {
+    final client = Matrix.of(context).client;
+    createNewGroupChatAction(
+      matrixClient: client,
+      createNewGroupChatRequest: CreateNewGroupChatRequest(
+        groupName: groupName,
+        invite: getSelectedValidContacts(contactsList)
+          .map<String>((contact) => contact.matrixId!)
```

Comment:
what happen if `matrixId` is null?

Reply:
updated 8e1f5767c7b099c2c9828ee8c810bb91e591429d